src/makeCalculator.js

```diff
@@ -35,7 +35,44 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    get add() {
+      return '+';
+    },
+    get subtract() {
+      return '-';
+    },
+    get multiply() {
+      return '*';
+    },
+    get divide() {
+      return '/';
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    operate(callback, value) {
+      switch (callback) {
+        case '+':
+          this.result += value;
+          break;
+        case '-':
+          this.result -= value;
+          break;
+        case '*':
+          this.result *= value;
+          break;
+        case '/':
+          this.result /= value;
+          break;
+      }
```

Comment:
don't you think writing logic in corresponding methods would be simpler?
don't forget about default case